src/woo_publications/api/tests/mixins.py

```diff
@@ -0,0 +1,181 @@
+from rest_framework import status
+from rest_framework.test import APIClient, APITestCase
+
+from .factories import TokenAuthFactory
+
+AUDIT_HEADERS = {
+    "AUDIT_USER_REPRESENTATION": "username",
+    "AUDIT_USER_ID": "id",
+    "AUDIT_REMARKS": "remark",
+}
+
+
+class APIKeyUnAuthorizedMixin:
+
+    self: APITestCase
```

Comment:
thought I had on Friday evening: this does nothing, as it would refer to `self.self`, which doesn't make any sense. So I'm guessing removing this line will change nothing.